Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -369,8 +419,6 @@ private void AddModulePanes()
                 html.LoadHtml(sb.ToString());
                 CheckPermission(html);
                 InjectBlocks(page, html);
-                if (!PageManager.InjectEditor(PortalSettings))
```

Comment:
Please merge upstream develop branch for change of this line

Reply:
Upstream develop branch has been merge